mods/adblock.js

```diff
@@ -152,28 +154,58 @@ function deArrowify(items) {
                 width: 1280,
                 height: 640
               }
-            ]
+            ];
+          } else {
+            item.tileRenderer.header.tileHeaderRenderer.thumbnail.thumbnails = [
+              {
+                url: fallbackThumbnail,
+                width: 320,
+                height: 180
+              }
+            ];
           }
         }
+      }).catch(() => {
+        item.tileRenderer.header.tileHeaderRenderer.thumbnail.thumbnails = [
+          {
+            url: fallbackThumbnail,
+            width: 320,
+            height: 180
+          }
+        ];
```

Comment:
Why are you replacing the thumbnail with nothing? It also doesn't need to be replaced here.